docs/getting-started/new_project_guide.md

```diff
@@ -524,3 +524,7 @@ Adding it is super easy, just follow this template:
 ```markdown
 [![Fuzzing Status](https://oss-fuzz-build-logs.storage.googleapis.com/badges/<project>.svg)](https://bugs.chromium.org/p/oss-fuzz/issues/list?sort=-opened&can=1&q=proj:<project>)
 ```
+
+## Monitoring performance via Fuzz Introspector
+
+As soon as your project is run with clusterfuzz, you can check the Fuzz Introspector report for your project. [Fuzz introspector](https://github.com/ossf/fuzz-introspector) is a tool to help you understand your fuzzer’s performance and identify any potential blockers. It provides individual and aggregated fuzzer reachability and coverage reports. You can monitor each fuzzer's static reachability potential and compare it against dynamic coverage and identify any potential bottlenecks. Fuzz Introspector can improve your fuzzer's performance by guiding you to decide whether to add a new fuzz target or modify an existing one to improve the quality or your harness. The report is accessible through the OSS-Fuzz report page for the C/C++ projects. Support for Java and Python projects is in the pipeline. As an example you can check Fuzz Introdpector [report for bzip2](https://storage.googleapis.com/oss-fuzz-introspector/bzip2/inspector-report/20221017/fuzz_report.html).
```

Comment:
1. s/clusterfuzz/ClusterFuzz (&lt; 1 day)
2. typo: Introdpector
3. s/The report is accessible through the OSS-Fuzz report page for the C/C++ projects./The report is accessible through the [OSS-Fuzz report page](https://oss-fuzz.com/) for C/C++ projects or through this [index](http://oss-fuzz-introspector.storage.googleapis.com/index.html).